testsuite/domain/src/test/resources/host-configs/host-slave.xml

```diff
@@ -141,6 +141,8 @@
         </server>
         <server name="reload-two" group="reload-test-group"  auto-start="false">
             <jvm name="default" />
+            <!-- this is for ServerRestartRequiredTestCase.testSocketBinding() unless the socket-binding-group is actually used for something it is ignored on the slaves, this is a quick fix before correcting the test -->
+            <socket-bindings socket-binding-group="reload-sockets" port-offset="1550"/>
```

Comment:
We need to correct this problem as part of this work. Unless my understanding of what is going on with this test is incorrect, this isn't a case of a incorrecdt test, it's the software itself that is not working correctly.

AIUI, in the domain config this test uses (domain-standard.xml), the "reload-sockets" s-b-g is used by the "reload-test-group" server group. Therefore this "reload-two" server-config means the "reload-sockets" is used by this server even without this element.